tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -50,15 +50,15 @@
 
 def make_partition(data, partition_index, shard_num):
   """
-    Shard keys to shard_num partitions
```

Comment:
Maybe use an independent commit to fix the format.

Reply:
Accept